wagtailmenus/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 # major.minor.patch.release.number
 # release must be one of alpha, beta, rc, or final
-VERSION = (3, 0, 1, "final", 0)
+VERSION = (3, 0, 2, "final", 0)
```

Comment:
Could you change "final" to "alpha" here for the time being? There may be other small things that we can fit into this release too.